src/lib/core/components/visualizations/implementations/BoxplotVisualization.tsx

```diff
@@ -0,0 +1,448 @@
+                  variable: vizConfig.overlayVariable,
+                },
+              ]}
+            />
+          </div>
+        </>
+      ) : (
+        // thumbnail/grid view
+        <Boxplot
+          data={data.value && !data.pending ? data.value.series : []}
+          containerStyles={{
+            width: 230,
+            height: 165,
+          }}
+          orientation={'vertical'}
+          // show/hide independent/dependent axis tick label
+          showIndependentAxisTickLabel={false}
+          showDependentAxisTickLabel={false}
+          showMean={true}
+          interactive={false}
+          displayLegend={false}
+          displayLibraryControls={false}
+          // margin is replaced with spacingOptions
+          spacingOptions={{
+            marginTop: 20,
+            marginRight: 20,
+            marginBottom: 0,
+            marginLeft: 30,
+          }}
+          showSpinner={data.pending}
+        />
+      )}
+    </div>
+  );
+}
+
+type BoxplotWithControlsProps = BoxplotProps;
+
+function BoxplotWithControls({
+  data,
+  ...BoxplotComponentProps
+}: BoxplotWithControlsProps) {
+  // TODO Use UIState
+  const errorManagement = useMemo((): ErrorManagement => {
+    return {
+      errors: [],
+      addError: (_: Error) => {},
+      removeError: (_: Error) => {},
+      clearAllErrors: () => {},
+    };
+  }, []);
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column' }}>
+      <Boxplot
+        {...BoxplotComponentProps}
+        data={data}
+        // add controls
+        displayLibraryControls={false}
+      />
+      {/* potential BoxplotControls: commented out for now  */}
+      {/* <BoxplotControls
+          // label="Box Plot Controls"
+          errorManagement={errorManagement}
+        /> */}
+    </div>
+  );
+}
+
+/**
+ * Reformat response from Box Plot endpoints into complete PromiseBoxplotData
+ * @param response
+ * @returns PromiseBoxplotData
+ */
+export function boxplotResponseToData(
+  response: PromiseType<ReturnType<DataClient['getBoxplot']>>
+): PromiseBoxplotData {
+  return {
+    series: response.boxplot.data.map(
+      (data: { [key: string]: any }, index) => ({
+        lowerfence: data.lowerfence,
+        upperfence: data.upperfence,
+        q1: data.q1,
+        q3: data.q3,
+        median: data.median,
+        mean: data.mean ? data.mean : undefined,
+        outliers: data.outliers ? data.outliers : undefined,
+        // currently returns seriesX and seriesY for points: 'all' option
+        // it is necessary to rely on rawData (or seriesX/Y) for boxplot if points: 'all'
+        rawData: data.rawData ? data.rawData : undefined,
+        // this will be used as legend
+        name: data.overlayVariableDetails
+          ? data.overlayVariableDetails.value
+          : 'Data',
+        // this will be used as x-axis tick labels
+        label: data[response.boxplot.config.xVariableDetails.variableId],
```

Comment:
I notice that the API docs and API response are not in agreement (sorry if I've missed this discussion elsewhere).

The `io-ts` validation is currently not validating this important part of the response, because the object key is different for different requests.  Here's a typical response (the third item in the reponse.boxplot.data array)  The key in this case is `EUPATH_0000096.PATO_0000047`.  

```
{
	"2": {
		"overlayVariableDetails": {
			"variableId": "PCO_0000024.EUPATH_0000730",
			"entityId": "PCO_0000024",
			"value": "Less frequent than a few times per week"
		},
		"EUPATH_0000096.PATO_0000047": [
			"Female",
			"Male"
		],
		"min": [
			0,
			1
		],
		"q1": [
			5,
			8
		],
		"median": [
			10,
			14
		],
               // and so on
	}
}
```

Should we go ahead with this PR as is, and make a ticket that the response should really have an array of strings something like this

```
boxplot.data[].xVariableValues  : array
boxplot.data[].xVariableValues[]  : string
```
?